bundles/pubsub/pubsub_spi/include/pubsub_protocol.h

```diff
@@ -28,39 +28,78 @@
 
 typedef struct pubsub_protocol_header pubsub_protocol_header_t;
 
+/**
+ * The protocol header structure, contains the information about the message payload and metadata
+ */
 struct pubsub_protocol_header {
-    unsigned int msgId;
-    unsigned short msgMajorVersion;
-    unsigned short msgMinorVersion;
-
-    unsigned int payloadSize;
-    unsigned int metadataSize;
+  /** message payload identification attributes */
+    unsigned int msgId; /*!< Message id of the payload */
```

Comment:
Curious about this notation: !< ... does it have some meaning? If not, does it make sense to clean this up? msgId with a comment "message id" etc doesn't add much imo.

Reply:
The notation is from Doxygen: http://www.doxygen.nl/manual/docblocks.html#memberdoc

Reply:
Even so, I don't see much value in any kind of doc that just re-iterates what the member already specifies.